activestorage/active.py

```diff
@@ -1,11 +1,16 @@
 import os
+import os.path
 import numpy as np
+import pathlib
+import s3fs
 
 #FIXME: Consider using h5py throughout, for more generality
 from netCDF4 import Dataset
 from zarr.indexing import (
     OrthogonalIndexer,
 )
+from activestorage.config import *
```

Comment:
it'd be good if this was renamed to `config_s3` so we can plug in other configurations too (eg maybe POSIX) - also, would you be able to make it a yaml file instead so it can be loaded at runtime as any dictionary, rather than an importable module? Makes things a bit more easier to maintain

Reply:
cheers for the changes @markgoddard :beer: And for the pointers how to do the tests. Let's do it this way then - let's get the conf as a Py module for now, and then I'll work out a yaml configuration in a separate PR, how's that sound to you? We already have https://github.com/valeriupredoi/PyActiveStorage/pull/47 that actually introduces exactly yaml-based configuration, so I can plug that there

Reply:
Yes, that makes sense - one step at a time.